src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```

Comment:
1. Here, you create three Lottery objects instead of Ball ones.
You should to create one Lottery, and using getRandomBall() create balls.
2. Bad naming. "ball1" but this is not actually the Ball instance.
3. Unnecessary invoke of toString() method because in println method of PrintStream instance(out) we invoking overrided toString() method of given argument object.
4. To create 3 random balls and print them you can use for loop.

`Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
 `